examples/nextjs-with-typescript/pages/MuxPlayer.tsx

```diff
@@ -137,25 +137,25 @@ const toValueString = (value: any) => {
 
 const MuxPlayerCodeRenderer = ({ state, component = 'MuxPlayer' }: { state: Partial<MuxPlayerProps>; component?: string; }) => {
   const stateEntries = Object.entries(state).filter(([,value]) => value != undefined);
-  const propsStr = stateEntries.length 
+  const propsStr = stateEntries.length
     ? `\n${stateEntries.map(([key, value]) => `  ${key}={${toValueString(value)}}`).join('\n')}\n`
     : '';
   const codeStr = `<${component}${propsStr}/>`;
-  const copyToClipboard = () => { 
-    navigator.clipboard?.writeText(codeStr); 
+  const copyToClipboard = () => {
+    navigator.clipboard?.writeText(codeStr);
   };
   return (
-    <div style={{ backgroundColor: 'lightgrey', display: 'flex', flexDirection: 'column', alignItems: 'flex-start' }}>
```

Comment:
The two big changes in this file are here to swap to a class and put the styles in css.